### ISSUE TITLE
Authenticating with 'client_credentials' throws 400 error 'invalid_scope'

### ISSUE THREAD

**Issue by ariestikto** [keycloak/keycloak-nodejs-admin-client#99]
**Describe the bug**
Calling `kcAdminClient.auth()` with `grantType: 'client_credentials'` throws 400 error 'invalid_scope'

**To Reproduce**
1. run kcAdminClient.auth()
```
await kcAdminClient.auth({
    clientSecret: xxx,
    grantType: 'client_credentials',
    clientId: xxx
});
```
2. It'll throw 400 error with message
```
{ 
    error: 'invalid_scope',
    error_description: 'Invalid scopes: ' 
}
```

**Expected behavior**
Keycloak should authorise the client. In version 1.13.0 its working as expected, upgrading to 1.14.2 show this behaviour

**Enviroment (please complete the following information):**
 - OS: Ubuntu 19.04
 - Keycloak Version: 10.0.2
 - Library Version 1.14.2

**Additional context**
Hasn't tried username + password authentication

**Comment by danazkari** [comment]
The culprit seems to be this 

https://github.com/keycloak/keycloak-nodejs-admin-client/commit/977e2324283899e66fbdc036fbfba751cfa61734#diff-79c8101c1f2ac59b33a8d000f8f14e6182042baf54f79df35d2151d3b4064405R49

They're adding `scope` empty and that seems to be breaking things

![image](https://user-images.githubusercontent.com/349542/100255836-a850a280-2f09-11eb-831c-73a3925b8a29.png)

**Comment by danazkari** [comment]
@ariestikto what did so far to get this working for me and to get unblocked was to add `offlineToken: true` to my credentials object like so:
![image](https://user-images.githubusercontent.com/349542/100259642-0d0dfc00-2f0e-11eb-82b7-8da68e9c7734.png)
And that seemed to do the trick, it includes a proper `scope` value instead of an empty one

I added a PR in the meantime #100 as an attempt to get this sorted out.

**Comment by ariestikto** [comment]
Hi @danazkari, that works thanks! well done for spotting the issue and thanks for the PR as well

**Comment by danazkari** [comment]
My absolute pleasure, this is a pretty good project and I like contributing whenever I can to open source projects I use daily, this was a very low hanging fruit, hopefully the fix makes sense

**Comment by edewit** [comment]
Thanks @danazkari also going back to version `1.14.1` would be an option

**Comment by danazkari** [comment]
@edewit thanks for the quick response! Lemme know if the PR is missing something

**Comment by edewit** [comment]
merged 2663b8427b77608200c3685772186f4cd1d7a74c
